.github/workflows/lint-test.yml

```diff
@@ -125,6 +125,10 @@ jobs:
         env:
           CYPRESS_TAKE_SNAPSHOTS: true
 
+      - name: Require approval when new snapshots are created 🔎
+        uses: numtide/clean-git-action@v1
+        if: success()
```

Comment:
This is not technically required, as it is the default - i.e. if a step fails, the following steps don't run. In this case, I want to make it clearer that this is sort of an `if/else` with the step below.